rst_files/lake_model.rst

```diff
@@ -217,60 +217,76 @@ Setup
 
 .. code-block:: julia
 
-    function LakeModel(;λ = 0.283, α = 0.013, b = 0.0124, d = 0.00822)
+    LakeModel = @with_kw (λ = 0.283, α = 0.013, b = 0.0124, d = 0.00822)
+
+    function A_matrices(lm)
```

Comment:
I don't like the name.  

Maybe `transition_matrices` instead

Reply:
Done.